torch/jit/_recursive.py

```diff
@@ -291,7 +305,7 @@ def create_methods_from_stubs(concrete_type, stubs):
     defaults = [get_default_args(m.original_method) for m in stubs]
     concrete_type._create_methods(defs, rcbs, defaults)
 
-def create_script_module(nn_module, stubs_fn, share_types=True):
+def create_script_module(nn_module, stubs_fn, submodule_stubs_fn=None, share_types=True):
```

Comment:
Why not adding new kwarg as last argument? Adding new arg in the middle runs the risk of someone incorrectly passing `share_types` as positional arg and be parsed as kwarg.